documentation/extensions/extensions.md

```diff
@@ -0,0 +1,31 @@
+# General Extensions to the GlobalPlatform TEE Internal API
+
+This document describes the OP-TEE extensions introduced with respect to the GlobalPlatform TEE API Specifications v1.0.
+
+Specific extensions documentation are part of:
+* Cryptographic Extensions
+  * [Concatenation Key Derivation](crypto_concat_kdf.md)
+  * [HMAC Key Derivation](crypto_hkdf.md)
+  * [Public-Key Key Derivation](crypto_pbkdf2.md)
+
+
+# Cache Maintenance Support
```

Comment:
Not so sure.
My purpose was to avoid having too much doc fragmented, as this one is quite small.

@jbech-linaro @jenswi-linaro what do you think? No problem to do it either way.